spring-cloud-aws-secrets-manager/src/main/java/io/awspring/cloud/secretsmanager/SecretsManagerPropertySource.java

```diff
@@ -89,8 +89,11 @@ private void readSecretValue(GetSecretValueRequest secretValueRequest) {
 		}
 		catch (JsonParseException e) {
 			// If the secret is not a JSON string, then it is a simple "plain text" string
-			LOG.debug("Populating property retrieved from AWS Secrets Manager: " + secretValueResponse.name());
-			properties.put(secretValueResponse.name(), secretValueResponse.secretString());
+			String secretName = secretValueResponse.name().lastIndexOf("/") != 0
```

Comment:
Can secrets end with "/"? If so, i think `/secret/foobar/` would end with empty `secretName`. Maybe we can verify that in tests?

Reply:
They can. But now thinking should we even support that, not sure if it makes sense to have a `secretName` ending with `/`:
`/something/prod/certificate/` not sure about ending `/` as valid end of a `secretName`.
Even in AWS samples they always have `/something/secretName`. 
Wdyt?

Reply:
It's rather the question if its allowed on the AWS side to end with `/`. If it is either we have to support it or throw meaningful error message